packages/apps/frontera/src/routes/organizations/src/components/PersonasFlowProfile/PersonasFlowProfile.tsx

```diff
@@ -0,0 +1,41 @@
+import { Tag, TagLabel } from '@ui/presentation/Tag';
+import { getContainerClassNames } from '@ui/form/Select';
+import { Certificate02 } from '@ui/media/icons/Certificate02';
+
+import { MultiSelectFilter } from '../shared';
+
+export const PersonasFlowProfile = () => {
+  return (
+    <>
+      <div className='flex'>
+        <p className='font-semibold'> Tag contact with </p>
+        <Tag variant='subtle'>
+          <TagLabel>Solo RevOps</TagLabel>
```

Comment:
Is tag label supposed to be hardcoded

Reply:
this part of the code will not be released